app/controllers/file_controller.rb

```diff
@@ -74,6 +74,10 @@ def current_file
   end
 
   def cocina
-    @cocina ||= Cocina.find(params[:id])
+    @cocina ||= Cocina.find(params[:id], version)
+  end
+
+  def version
+    params[:version_id] || :head
```

Comment:
Should `version_id` also be permitted in `file_params` (so the bounce through auth can work)?

Reply:
Hmm, I'm not sure about `WebauthController` as it symbolizes the keys I think it should be good as is, but open to suggestion.

Reply:
Don't we need to check if there's a version parameter and send it to the right route? Right now, it looks like it will always redirect to the non-versioned path?

https://github.com/sul-dlss/stacks/blob/main/app/controllers/webauth_controller.rb#L32

Reply:
@cbeer ah I see, I thought that the params would determine that, but walking through a debugger I see it passes it as a query param not part of the path. Fixed.